tests/src/test/java/com/netflix/msl/msg/MessageBuilderSuite.java

```diff
@@ -843,13 +874,16 @@ public void deleteServiceToken() throws MslException {
         @Test
         public void deleteUnknownServiceToken() throws MslException {
             final MessageBuilder builder = messageFactory.createRequest(trustedNetCtx, MASTER_TOKEN, USER_ID_TOKEN);
-            builder.deleteServiceToken(SERVICE_TOKEN_NAME);
+            builder.deleteServiceToken(SERVICE_TOKEN_NAME, true, true);
             final MessageHeader messageHeader = builder.getHeader();
             final Set<ServiceToken> tokens = messageHeader.getServiceTokens();
             for (final ServiceToken token : tokens) {
-                if (token.getName().equals(SERVICE_TOKEN_NAME))
-                    fail("Deleted unknown service token.");
+                if (token.getName().equals(SERVICE_TOKEN_NAME)) {
+                    assertEquals(0, token.getData().length);
```

Comment:
This is a strange assert. Suggest putting in real data to confirm or else delete the assert.

Reply:
Confirming length == 0 is confirming that the "deletion token" was created properly.